pkg/cmd/cli/cmd/deploy.go

```diff
@@ -352,6 +361,9 @@ func (o DeployOptions) cancel(config *deployapi.DeploymentConfig, out io.Writer)
 
 // reenableTriggers enables all image triggers and then persists config.
 func (o DeployOptions) reenableTriggers(config *deployapi.DeploymentConfig, out io.Writer) error {
+	if config.Spec.Paused {
+		return fmt.Errorf("cannot enable triggers for a paused deployment config")
```

Comment:
Actually I just realized that automatic and a config change trigger can be whatever they want, paused supersedes them. I think it's better to supersede than try to align all values. I guess it's ok to remove the restriction here.